dace/sdfg/state.py

```diff
@@ -99,6 +99,10 @@ def in_degree(self, node: NodeT) -> int:
     def out_degree(self, node: NodeT) -> int:
         ...
 
+    @property
+    def sdfg(self) -> 'dace.sdfg.SDFG':
```

Comment:
you should probably import dace.sdfg.SDFG in the TYPE_CHECKING clause (line 28)

Reply:
Done